fuelup-init.sh

```diff
@@ -24,7 +24,13 @@ main() {
     mkdir -p "$FUELUP_DIR/bin"
 
     local _fuelup_version
-    _fuelup_version="$(curl -s https://api.github.com/repos/FuelLabs/fuelup/releases/latest | grep "tag_name" | cut -d "\"" -f4 | cut -c 2-)"
+    _published_fuelup_version_url="https://raw.githubusercontent.com/FuelLabs/fuelup/gh-pages/fuelup-version"
+    _fuelup_version="$(curl -s $_published_fuelup_version_url)"
+    if echo "$_fuelup_version" | grep -q -E '404|400'; then
```

Comment:
Would this condition be missed on a future version like `1.0.404`? I guess it's unlikely we'd reach that version :smiling_face_with_tear:  and it would just get fetched from the GitHub API anyway.